app/styles/utilities/_hide.scss

```diff
@@ -9,3 +9,9 @@
     display: none;
   }
 }
+
+@media (min-width: $Theme-break-mobile-nav) {
```

Comment:
As we discussed in person, this media query should be related to the `.Nav-burger` component.